### PR TITLE
Confluent: update confluent to new Spec (and slack)

### PR DESCRIPTION
**Pull request by odacremolbap**
- Update confluent bridges to use the new Spec.
- Also adapt Slack bridges

### CODE REVIEW

File: bridges/kinesis-confluent/README.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
 # Forward Kinesis messages to a Confluent Kafka cluster
 
-Create a secret containing your AWS API credentials
+This bridge listens to a kinesis stream and send received messages to a Confluent topic.
```
Comment (sebgoa) [inline bridges/kinesis-confluent/README.md:3]: and sends

File: bridges/slack-confluent/slack-confluent.yaml [hunk 2]
```diff
@@ -44,13 +44,11 @@ spec:
...
-              name: confluent
-              key: username
+          securityProtocol: SASL_SSL
+          saslMechanism: PLAIN
+          bootstrapServers:
+            - <CONFLUENT-HOST:CONFLUENT-PORT>
+          username: <USERNAME>
           password:
             secretKeyRef:
               name: confluent
```
Comment (sebgoa) [inline bridges/slack-confluent/slack-confluent.yaml:54]: key ?

Comment (odacremolbap) [reply bridges/slack-confluent/slack-confluent.yaml:54]: not sure if I follow.

The password for the user is at the next line:

https://github.com/triggermesh/bridges/blob/97101d6e5e85d7e3b7b18ca3620c68e8db237d7e/bridges/slack-confluent/slack-confluent.yaml#L52-L55

Comment (sebgoa) [reply bridges/slack-confluent/slack-confluent.yaml:54]: my bad, did not see the `key:` being set

**Pull request merged**